sites/friday/layouts/shortcodes/guides/deploy/configure/resources.md

```diff
@@ -0,0 +1,68 @@
+## Allocate preview resources
+
+In the [{{ .Site.Params.vendor.name }} Console]({{ .Site.Params.vendor.urls.console }}), you'll see two new things:
+
+- an _inactive_ environment for the branch you've committed to
+- an _active_ environment for the pull request you've opened
+
+There are other was to setup a source integration, but this is the default behavior.
+
+![Production environment](/images/pr-env.png ".4")
+
+In the **Activity** section, you'll see a failed activity:
+
+<div style="text-align: center">
+
+{{ printf `{{< icon failure >}}` | .Page.RenderString }} **GitHub integration** pushed to **PR #1 Add {{ .Site.Params.vendor.name }} configuration**
+
+</div>
+
+On the failed {{ printf `{{< icon failure >}}` | .Page.RenderString }} push activity, click on {{ printf `{{< icon more >}}` | .Page.RenderString }} **More**, then **View log**.
+At the end of the build and deploy phases, at the bottom of the logs, 
+you will see the activity failed with a message similar to the below:
+
+{{ .Inner }}
+
+You see this message because while you have pushed infrastructure configuration to {{ .Site.Params.vendor.name }}, {{ .Site.Params.vendor.name }} doesn't yet know how much disk, CPU, and memory to allocated to those containers. 
+
+Run the following command:
+
+```bash
+{{ .Site.Params.vendor.cli }} resources:set
+```
+
+Follow the prompts to configure and confirm resources for each of the containers in the environment.
```

Comment:
should we give them a link to more information explaining the resource sizes?

Reply:
and maybe give more info/advice telling the reader that minimum `0.1` can be used for each of the web/services?